_commands/basics/pwd.md

```diff
@@ -0,0 +1,49 @@
+---
+---
+
+pwd
+--
+
+`pwd` stands for print working directory. `pwd` outputs the full pathname of the current work director. 
+
+~~~ bash
+$ pwd
+/home/bo/Desktop
+~~~
+
+`pwd` can also be used to store the full path to the current directory. 
+
+~~~ bash
+x=$(pwd)
+~~~
+
+### Useful Options / Examples 
+
+#### `pwd -L`
+
+~~~ bash
+$ ln -s . test
+$ cd test && pwd
+/home/bo/Desktop/test
+$ /bin/pwd
+/home/bo/Desktop
+$ /bin/pwd -L
+/home/bo/Desktop/test
+~~~
+
+##### Break it down
```

Comment:
It's not clear from this where the "other" pwd is coming from. Could you add an explanation about that?